seed/challenges/01-front-end-development-certification/basic-javascript.json

```diff
@@ -2419,6 +2427,10 @@
         "",
         "// Only change code below this line",
         "",
+        "",
+        "// Only change code above this line",
+        "",
+        "timesFive(5);",
```

Comment:
@ChadKreutzer correct, the function call isn't needed to pass the challenge. However, the code as you have it will prevent the camper from knowing what they need to do and they will have to delete that line anyways. 

![image](https://cloud.githubusercontent.com/assets/2754821/16537799/800c06a6-3fc2-11e6-842f-caf59ffc1933.png)